protocol/contracts/libraries/Silo/LibWhitelist.sol

```diff
@@ -98,13 +99,38 @@ library LibWhitelist {
         verifyGaugePointSelector(gaugePointSelector);
         verifyLiquidityWeightSelector(liquidityWeightSelector);
 
-        // add whitelist status
-        LibWhitelistedTokens.addWhitelistStatus(
-            token,
-            true, // Whitelisted by default.
-            token != address(C.bean()) && !LibUnripe.isUnripe(token), // Assumes tokens that are not Unripe and not Bean are LP tokens.
-            selector == LibWell.WELL_BDV_SELECTOR
-        );
+        // fetch the whitelist status of the token. Brackets due to stack too deep.
```

Comment:
Maybe worth considering pulling this out into it's own function to handle the previously whitelisted case, but the brackets are fine with me. Overall I couldn't find any problems with the remediations. Super minor complaint about `if(` with no space before the `(` but not worth messing with.